### PR TITLE
Add podcast skip buttons

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Move next/prev buttons to providers and add podcast skip buttons

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
Nah

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.
* Locally against prod/staging
* Local mobile web + apps

### CODE REVIEW

File: src/containers/now-playing/NowPlaying.tsx [hunk 1]
```diff
@@ -292,6 +294,15 @@ const NowPlaying = g(
     ])
 
     const onPrevious = () => {
+      if (track.genre === Genre.PODCASTS) {
+        const position = timing.position
+        const newPosition = position - SKIP_DURATION_SEC
+        if (newPosition > 0) {
```
Comment (sddioulde) [inline src/containers/now-playing/NowPlaying.tsx:300]: should we use seek(Math.mix(0, newPosition)) and not do this check

File: src/containers/now-playing/NowPlaying.tsx [hunk 2]
```diff
@@ -300,6 +311,17 @@ const NowPlaying = g(
       }
     }
 
+    const onNext = () => {
+      if (track.genre === Genre.PODCASTS) {
+        const newPosition = timing.position + SKIP_DURATION_SEC
+        if (newPosition < timing.duration) {
```
Comment (sddioulde) [inline src/containers/now-playing/NowPlaying.tsx:317]: here too, should we use seek(Math.min(newPosition, timing.duration)) and not do this check

File: src/containers/play-bar/desktop/PlayBar.js [hunk 3]
```diff
@@ -224,12 +226,46 @@ class PlayBar extends Component {
...
+      audio,
+      seek,
+      previous,
+      reset,
+      currentQueueItem: { track }
+    } = this.props
+    if (track.genre === Genre.PODCASTS) {
+      const position = audio.getPosition()
+      const newPosition = position - SKIP_DURATION_SEC
+      if (newPosition > 0) {
```
Comment (sddioulde) [inline src/containers/play-bar/desktop/PlayBar.js:239]: here too

File: src/containers/play-bar/desktop/PlayBar.js [hunk 4]
```diff
@@ -224,12 +226,46 @@ class PlayBar extends Component {
...
+      audio,
+      seek,
+      next,
+      currentQueueItem: { track }
+    } = this.props
+    if (track.genre === Genre.PODCASTS) {
+      const duration = audio.getDuration()
+      const position = audio.getPosition()
+      const newPosition = position + SKIP_DURATION_SEC
+      if (newPosition < duration) {
```
Comment (sddioulde) [inline src/containers/play-bar/desktop/PlayBar.js:263]: and here

File: src/containers/now-playing/NowPlaying.tsx (hunk 1 above)
Comment (raymondjacobson) [reply src/containers/now-playing/NowPlaying.tsx:300]: subtle difference -- if newPosition is <= 0, instead, we want to not return and fall through, so i think this is best. thoughts?

Comment (raymondjacobson) [reply src/containers/now-playing/NowPlaying.tsx:300]: nvm u right

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/rj-aud-588

**Review by sddioulde — APPROVED** [review]

**Pull request merged**